bin/dev/legalscan.sh

```diff
@@ -1,4 +1,9 @@
 #!/usr/bin/env bash
+
+for PACKAGE in $(osc ls Cloud:Platform:buildpacks:dependencies); do
+  osc submitrequest --yes --message=legal-review Cloud:Platform:buildpacks:dependencies $PACKAGE Cloud:Platform:sources:buildpacks:reviewed
```

Comment:
IMHO it would make sense to move the project names to variables.

Reply:
At the moment we only use them in one script. What variable names would you suggest?

Reply:
`SOURCE_OBS_PROJECT` and maybe `TARGET_OBS_PROJECT`